app/src/test/java/com/aws/amazonlocation/utils/turf/TurfMetaCoordsPointTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.aws.amazonlocation.utils.turf
+
+import com.aws.amazonlocation.mock.TEST_DATA_LAT
+import com.aws.amazonlocation.mock.TEST_DATA_LNG
+import com.aws.amazonlocation.utils.geofence_helper.turf.TurfMeta
+import com.mapbox.geojson.MultiLineString
+import com.mapbox.geojson.Point
+import com.mapbox.geojson.Polygon
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class TurfMetaCoordsPointTest {
+
+    @Test
+    fun turfMetaCordPolygonLatLngTest() {
+        val pointList = TurfMeta.coordAll(
+            Polygon.fromLngLats(
+                mutableListOf(
+                    mutableListOf(
+                        Point.fromLngLat(
+                            TEST_DATA_LAT,
+                            TEST_DATA_LNG
+                        )
+                    )
+                )
+            ), false
+        )
+        Assert.assertTrue(pointList.isNotEmpty())
```

Comment:
What is the purpose of this test? It adds a latlon into the pointList and assert the point list is not empty. isn't it always passed.